Achievements/SelfMade.lua

```diff
@@ -1,14 +1,15 @@
 local _G = _G
 local self_made_achievement = CreateFrame("Frame")
 _G.achievements.SelfMade = self_made_achievement
+Player = UnitName("Player")
```

Comment:
Should make local, this has a high chance of clashing somewhere elsen in the addon

Reply:
attempted to make it local but for some reason it was not pulling the player name to check it. I will do some local testing see where I can make it work, i didnt want to go Global as that usually ends up bad.

Reply:
Could need to be UnitName("player") (uncaps)

Reply:
i was able to get it working by adding the local player = UnitName("player") to each of the functions inside the loop and the initial function call.